From 77e86991f21c895a8907e2b900e6ff776206e978 Mon Sep 17 00:00:00 2001
From: Tao Guo <tao.guo@amlogic.com>
Date: Thu, 22 Apr 2021 13:46:31 +0800
Subject: [PATCH] nrdp: fix av1 support api [1/1]

PD#SWPL-47067

Problem:
S905X2 NTS can not pass.
nrdp report support AV1, so netflix send 4.3 headers
but playready 3.3 not support 4.3 header
to support AV1, playready need upgrade to 4.0+

Solution:
fix AV1 support api

Verify:
S905x2 + RDK

Change-Id: I45783af16b875fe9a2be5f7b13fb01c12762c9cf
Signed-off-by: Tao Guo <tao.guo@amlogic.com>
---
 partner/dpi/gstreamer/AmlUtils.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/partner/dpi/gstreamer/AmlUtils.cpp b/partner/dpi/gstreamer/AmlUtils.cpp
index 89ea632b..6e9df168 100644
--- a/partner/dpi/gstreamer/AmlUtils.cpp
+++ b/partner/dpi/gstreamer/AmlUtils.cpp
@@ -442,9 +442,9 @@ bool IsHDMIDVSupported(){
 
 bool Aml_IsSupportAV1()
 {
-    return ((Aml_Get_Soc_Type() == (AML_SOC_TYPE_S805) ||
-                                  (AML_SOC_TYPE_S805X2)||
-                                  (AML_SOC_TYPE_S905X4 )) ? (bool)true : (bool)false);
+    return ((Aml_Get_Soc_Type() == (AML_SOC_TYPE_S908X) ||
+            Aml_Get_Soc_Type() == (AML_SOC_TYPE_S805X2)||
+            Aml_Get_Soc_Type() == (AML_SOC_TYPE_S905X4)) ? (bool)true : (bool)false);
 }
 
 bool Aml_IsSeries8()
-- 
2.31.1

